calculate.py

```diff
@@ -21,12 +26,49 @@ def evaluate(expression: str) -> str:
         expression = re.sub('(?![a-zA-Z])e(?![a-zA-Z])', 'E', expression)
         expression = expression.replace('π', 'PI')
         expression = expression.replace('√', 'sqrt')
+
+        # Convert numbers directly to rad
+        if exp_is_only_num(expression):
+            if is_rad == "true":
+                expression = str(display.rad((float)(expression)))
+            elif is_rad == "false":
+                expression = str(display.deg((float)(expression)))
+
         # Evaluate expression
         return parser.evaluate(expression)
     except Exception as e:
         return str(e)
 
 
+def exp_is_only_num(expression: str) -> bool:
+    """Evaluates mathematical expression passed to determine if it is a number (float or int)
+
+    Args:
+        expression (str): Expression to evaluate
+
+    Returns:
+        bool: Result of evaluation of expression type
+    """
+    try:
+        float(expression)
+        return True
+    except ValueError:
+        return False
+
+
+def exp_is_blank(expression: str) -> bool:
+    """Evaluates mathematical expression passed to determine if it is empty
+
+        Args:
+            expression (str): Expression to evaluate
+
+        Returns:
+            bool: Result of evaluation of expression length
+        """
+    return not (expression and expression.strip())

```

Comment:
Your code works either way here so this isn't really important but why not just check for the value of expression.strip()? Isn't it the same as expression when the string is blank?

Reply:
Unless it's supposed to be an optimization?

Reply:
Hmm, I was thinking in Java. The rationale was if expression is null, we shouldn't be calling a function on it. 
You might be right about this not being necessary in Python. 
I'll verify it behaves as expected next PR and remove it.

Reply:
Ahh okay I wasn't thinking along those lines. I guess that calling strip on a null value would throw an exception.